src/components/Quotes.js

```diff
@@ -44,7 +44,7 @@ function Quotes() {
   return (
     <div className="qoute-cont">
       <p>{quoteData && quoteData.quote}</p>
-      <h2>{quoteData.author}</h2>
+      <h2 className="aouthor">{quoteData.author}</h2>
     </div>
   );
```

Comment:
- [ ] Please make sure that the general layout of your `Quotes` page matches the given wireframe in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md).  As you can see in the wireframe the quote is center aligned

|Your design |wireframe design|
|------|------|
|<img width="1286" alt="Screenshot 2023-06-14 at 9 30 01 AM" src="https://github.com/rabayarabu/calculator/assets/44798044/d52f99b5-89d3-4591-8c6b-75fc76b637f3">|<img width="886" alt="Screenshot 2023-06-14 at 9 30 26 AM" src="https://github.com/rabayarabu/calculator/assets/44798044/5959d4e1-d079-4a11-8fe0-4da5648fea2a">|

**NOTE:** Students are encouraged to add their styles to improve the look and feel.